app-config.yaml

```diff
@@ -48,6 +48,9 @@ auth:
         clientSecret:
           $secret:
             env: AUTH_GITHUB_CLIENT_SECRET
+        enterpriseInstanceURL: 
```

Comment:
Can I be annoying and request that we camel-case it as `enterpriseInstanceUrl`? :grin: Keeps it similar to `baseUrl` etc above. I know the passport config requires `URL` though, so keep the vars below the same.